src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,9 @@
+$days-in-week: 7;
+$days-in-month: 31;
+$day-width: 100px;
+$week-width: $day-width * $days-in-week;
+$month-padding: 10px;
+$background-color: #eee;
+$color-black: #000;
+$color-pink: #FFBFCB;
+$weekdays: 1 "mon", 2 "tue", 3 "wed", 4 "thu", 5 "fri", 6 "sat", 7 "sun";
```

Comment:
you have the "sun" here, and the app looks broken with that modifier
<img width="1356" alt="image" src="https://github.com/mate-academy/layout_calendar/assets/70220667/90ad7e9b-2f57-4477-bc27-7782196c51be">


please, watch this video, how your task should work, and don't hesitate to ask for some help in the chat

https://www.loom.com/share/3dcf1ba5147f45139ef2d1a0d340b0d3?sid=a813b58a-6789-4dd7-a7b7-ab53ba725dab